packages/firebase_auth/firebase_auth_platform_interface/lib/src/method_channel/utils/convert_auth_provider.dart

```diff
@@ -24,5 +24,11 @@ AuthProvider convertToOAuthProvider(AuthProvider authProvider) {
     return oAuthProvider;
   }
 
+  if (authProvider is TwitterAuthProvider) {
+    final oAuthProvider = OAuthProvider(authProvider.providerId);
```

Comment:
do we have to set scopes here? `oAuthProvider.setScopes(authProvider.scopes);`

Reply:
Twitter does not offer scopes, so it's not available on the TwitterAuthProvider